vars/common-vars.yml

```diff
@@ -125,3 +125,5 @@ scale_profile: "{{ lookup('env','scale_profile') | default('minimal', true) }}"
 
 #Set timeout for helm cleanup.
 helm_delete_timeout: 900
+
+airship_ucp_workers: 2
```

Comment:
The meaning is actually "control workers", right? As opposed to "compute workers", if I get it right. So it is a subset of "caasp workers" and current variable name does not make it clear IMHO